pbj-integration-tests/src/main/java/com/hedera/pbj/integration/fuzz/FuzzTest.java

```diff
@@ -37,6 +37,18 @@ public static <T> FuzzTestResult<T> fuzzTest(final T object, final double thresh
         final Random random = new Random();
```

Comment:
Please, print out the seed of this `Random` object.

Reply:
Random doesn't have a getter for its seed.
I'm working on creating a Random with a fixed seed right now, but that will come in a separate PR.